doc/references.bib

```diff
@@ -2077,3 +2075,17 @@ @article{YaoEtAl2019
   doi = {10.1007/s10548-019-00707-x},
   publisher={Springer}
 }
+
+@article{PolonenkoMaddox2019,
+  title = {The {Parallel} {Auditory} {Brainstem} {Response}},
+  volume = {23},
+  issn = {2331-2165},
+  doi = {10.1177/2331216519871395},
+  language = {en},
+  journal = {Trends in Hearing},
+  author = {Polonenko, Melissa J. and Maddox, Ross K.},
+  month = jan,
+  year = {2019},
+  note = {Publisher: SAGE Publications Inc},
```

Comment:
```suggestion
  year = {2019},
```
`month` and `note` fields actually show up in the rendered bibentry.  Hopefully some day we'll be able to provide a custom CSL to sphinxcontrib-bibtex (that doesn't print such things) so we won't have to curate our bibtex fields so carefully.